cinema_service/settings.py

```diff
@@ -102,15 +100,15 @@
     },
     {
         "NAME": "django.contrib.auth.password_validation."
-        "MinimumLengthValidator",
+                "MinimumLengthValidator",
```

Comment:
Why this was changed?

Reply:
Flake8 found a lot of place where not double quotes. And I decided to use "Black ."